pyodide_build/mkpkg.py

```diff
@@ -83,37 +121,45 @@ def update_package(package: str):
         print(f"Skipping: {package} is a local package!")
         sys.exit(0)
 
-    if set(yaml_content.keys()).difference(
-        ("package", "source", "test", "requirements")
-    ):
```

Comment:
Yes, this wasn't too reliable

Reply:
Well I kept this, I just moved it down because I want it to report whether the package is out of date or not even if it can't be updated.